applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```diff
@@ -144,6 +142,20 @@ ConstitutiveLaw::StressMeasure BaseSolidElement::GetStressMeasure() const
 /***********************************************************************************/
 /***********************************************************************************/
 
+void BaseSolidElement::SetConstituveLawFlags(
```

Comment:
I would call this function `SetDefaultConstituveLawFlags`, maybe this makes it more clear